src/Contributors/Model/Contributor.php

```diff
@@ -39,7 +39,7 @@ public function getProfileUrl(): string
 
     public function getProfileImg(): string
     {
-        return $this->profileImg.'?s='.$this->size;
+        return $this->profileImg;
```

Comment:
Is it not necessary?

Reply:
It's a known issue, for now it is a workaround.